index.js

```diff
@@ -0,0 +1,118 @@
+const tripsUrl = 'https://trektravel.herokuapp.com/trips';
+
+const displayStatus = (message) => {
+    $('#status').html(message);
+  };
+  
+const reportError = (message, errors) => {
+  let content = `<p>${message}</p><ul>`;
+  for (const field in errors) {
+    for (const problem of errors[field]) {
+      content += `<li>${field}: ${problem}</li>`;
+    }
+  }
+  content += "</ul>";
+  reportStatus(content);
```

Comment:
On line 15, do you mean `displayStatus`? `reportStatus` isn't defined in this file.